src/signals/incident-management/components/FilterForm/utils/has-too-many-filters-selected.ts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+
+import { generateParams } from 'shared/services/api/api'
+import CONFIGURATION from 'shared/services/configuration/configuration'
+
+import { MAX_FILTER_LENGTH } from './constants'
+
+export const hasTooManyFiltersSelected = (
+  storedParams: any,
```

Comment:
Dit kan getyped worden als KeyValuePair<"iets hierin">

Reply:
Ik weet niet of dit de hele lading dekt omdat ik ik niet zeker weet wat voor types alle opties kunnen hebben die geparsed wordt. Onderstaande zou ik eventueel kunnen doen, maar voegt dat iets toe als ik het niet 100% zeker weet? (FilterForm.js is niet getyped dus daar houd het alweer op qua type securty)
<img width="527" alt="Screenshot 2022-08-30 at 11 03 31" src="https://user-images.githubusercontent.com/46756331/187396595-6eb71971-a536-4b37-9144-827d912f181d.png">